Source/WebKit/Shared/WebPageCreationParameters.h

```diff
@@ -187,7 +187,7 @@ struct WebPageCreationParameters {
 #if USE(WPE_RENDERER)
     IPC::Attachment hostFileDescriptor;
 #endif
-#if PLATFORM(WIN)
+#if PLATFORM(WIN) || USE(GRAPHICS_LAYER_WC)
     uint64_t nativeWindowHandle;
```

Comment:
Is `nativeWindowHandle` required for WC?

I don't think all the plumbing is there for PlayStation if so.

Reply:
Yes. `nativeWindowHandle` is used for `GLContext::createContextForWindow`.
https://github.com/WebKit/WebKit/blob/12f417282701d97084722aa8cad975f458fd66e4/Source/WebKit/GPUProcess/graphics/wc/WCSceneContext.cpp#L38
I have a plan to pass the `nativeWindowHandle` from UI process to GPU process directly in the future.
For the security, I don't want to tell web process the `nativeWindowHandle`.